packages/dd-trace/src/exporters/browser/index.js

```diff
@@ -55,6 +52,16 @@ class BrowserExporter {
       this._flushing = false
     })
   }
+
+  _traceMeta () {
+    const meta = {
+      '_dd.source': 'browser'
+    }
+
+    addTag(meta, 'env', this._env)
```

Comment:
Why bother having this function? Why not just have:

```js
if (this._env) {
  meta.env = this._env
}
```

Reply:
It will be less code when adding new tags since we won't need many ifs. It also moves the concern of how to add a tag outside of this function.